example/lib/main.dart

```diff
@@ -1 +1,7 @@
-void main() {}
+void info() {
+  final unused = ''
+}
+
+void error() {
+  return null;
```

Comment:
<table>
  <tbody>
    <tr>
      <td>:rotating_light:</td>
      <td>Avoid returning null for void.</td>
    </tr>
  </tbody>
</table>